mobile/lib/features/trade/trade_bottom_sheet_tab.dart

```diff
@@ -350,8 +350,13 @@ class _TradeBottomSheetTabState extends State<TradeBottomSheetTab>
                     selector: (_, provider) =>
                         provider.fromDirection(direction).liquidationPrice ?? 0.0,
                     builder: (context, liquidationPrice, child) {
-                      return ValueDataRow(
-                          type: ValueType.fiat, value: liquidationPrice, label: "Liquidation:");
+                      if (tradeValues.openQuantity == tradeValues.contracts) {
```

Comment:
How does it know that this is a close? I mean, wouldn't this also be true if you are `100 long` and then go another `100 long`?

Reply:
`openQuantity` is only set for the opposite direction. So if you'd go 100 long and then again 100 long the open quantity would be 0.

It would be 100 for the short direction. So if you'd go 100 short, you'd close the position.

Reply:
oh man, this really needs to be documented somewhere 🙈

Reply:
I will extend the docs on the open quantity field